src/ConvertToSarif/ConvertToSarif.csproj

```diff
@@ -7,6 +7,15 @@
     <AssemblyName>ConvertToSarif</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <Prefer32Bit>true</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <Prefer32Bit>true</Prefer32Bit>
+    <PlatformTarget>x86</PlatformTarget>
```

Comment:
No that change was to avoid warnings about enabling ?SAFESEH for x64 build

---
In reply to: [134349989](https://github.com/Microsoft/sarif-sdk/pull/590#discussion_r134349989) [](ancestors = 134349989,134336111)